component/modal/modal.css

```diff
@@ -0,0 +1,16 @@
+.modal-container{
+    position: relative;
+}
+.modal{
+    position: absolute;
+    background-color: var(--white);
```

Comment:
observed that usage of colors like white and black with shades for text and backgrounds are common so we will use at many places so can't we create one such css variable and use whereever it requires,
here you have used same background color in **modal** class and for **modal btn**